glyph/src/ttfp.rs

```diff
@@ -19,23 +19,51 @@ impl From<GlyphId> for ttfp::GlyphId {
 
 /// A pre-rendered image of a glyph, usually used for emojis or other glyphs
 /// that can't be represented only using an outline.
+#[deprecated(since = "0.2.21", note = "Deprecated in favor of `GlyphImage2`")]
```

Comment:
Since should be the _next_ version

```suggestion
#[deprecated(since = "0.2.22", note = "Deprecated in favor of `GlyphImage2`")]
```